<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XCalc</class>
 <widget class="QMainWindow" name="XCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calc X BlackWood Peregrin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #3d3d3d;</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QTextEdit" name="textEdit_formule">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>75</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid rgb(12, 0, 0);
font: 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;
border-radius: 20px;
background-color: rgba(90, 16, 180, 110);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Comic Sans MS'; font-size:20px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_input">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid gray;
border-radius: 20px;
background-color: black;
color: white;
font-size: 25px;
font-family: &quot;Comic Sans MS&quot;;

</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_result">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>75</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font:  20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;
border: 2px solid black;
border: 2px solid gray;
border-radius: 20px;
background-color: rgba(90, 16, 180, 110);
background-color: black;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Comic Sans MS'; font-size:20px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20px;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <property name="horizontalSpacing">
        <number>8</number>
       </property>
       <property name="verticalSpacing">
        <number>15</number>
       </property>
       <item row="1" column="5">
        <widget class="QPushButton" name="but_6">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 25px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(90, 16, 180, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <widget class="QPushButton" name="but_equal">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 25px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(70, 12, 190, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="but_4">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
width: 40px;
height: 45px;
border-radius: 25px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(139, 23, 167, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="but_clear_last">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 25px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(103, 10, 166, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>&lt;--</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QPushButton" name="but_3">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 25px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background:  rgba(80, 16, 183, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="but_5">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 25px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(103, 10, 166, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="but_0">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 25px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(90, 16, 180, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="but_clear">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 25px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(175, 48, 173, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>AC</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QPushButton" name="but_dot">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 25px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background:  rgba(80, 16, 183, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="but_plus">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
width: 40px;
height: 45px;
border-radius: 25px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(175, 48, 173, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="but_minus">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 25px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(139, 23, 167, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="but_1">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
width: 40px;
height: 45px;
border-radius: 25px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(103, 10, 166, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="but_8">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
width: 40px;
height: 45px;
border-radius: 25px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(139, 23, 167, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QPushButton" name="but_2">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 25px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(90, 16, 180, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="but_9">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 25px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(103, 10, 166, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="but_7">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 25px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(175, 48, 173, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionfdsfds">
   <property name="text">
    <string>fdsfds</string>
   </property>
  </action>
  <action name="actionfdsfds_2">
   <property name="text">
    <string>fdsfds</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
